pages/join/form.tsx

```diff
@@ -18,7 +33,7 @@ UpdateProfile.defaultProps = {
   initialValues: undefined,
 };
 
-UpdateProfile.getInitialProps = async ctx => {
+UpdateProfile.getInitialProps = async (ctx: NextPageContext): Promise<UpdateProfileProps> => {
```

Comment:
check out docs on Next.js and TypeScript. There should be a type that we can verify via:

`UpdateProfile.getInitialProps: SomeTypeHere` and then everything else will work fine. I forget the name of it!